File: DarkOrbit 10.0/Net/netty/handlers/LegacyModuleHandler.cs
using Ow.Game;
using Ow.Managers;
using Ow.Net.netty.commands;
using Ow.Net.netty.requests;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ow.Net.netty.handlers
{
    class LegacyModuleHandler : IHandler
    {
        public void execute(GameSession gameSession, byte[] bytes)
        {
            var read = new LegacyModuleRequest();
            read.readCommand(bytes);

            var player = gameSession.Player;
            string[] packet = read.message.Split('|');
            switch (packet[0])
            {
                case ServerCommands.SET_STATUS:
                    switch (packet[1])
                    {
                        case ClientCommands.CONFIGURATION:
                            player.ChangeConfiguration(packet[2]);
                            break;
                    }
                    break;
            }
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/handlers/LegacyModuleHandler.java
package simulator.netty.handlers;

import net.ClientCommands;
import net.ServerCommands;
import net.game_server.GameServerClientConnection;
import net.game_server.GameSession;

import simulator.netty.ClientCommand;
import simulator.netty.clientCommands.LegacyModuleRequest;
import utils.Log;

/**
 Created by Pedro on 31-03-2015.
 */
public class LegacyModuleHandler
        implements ICommandHandler {

    private final GameServerClientConnection mGameServerClientConnection;
    private final LegacyModuleRequest        mLegacyModuleRequest;

    public LegacyModuleHandler(final GameServerClientConnection pGameServerClientConnection,
                               final ClientCommand pLegacyModuleRequest) {
        this.mGameServerClientConnection = pGameServerClientConnection;
        this.mLegacyModuleRequest = (LegacyModuleRequest) pLegacyModuleRequest;
    }

    @Override
    public void execute() {
        Log.pt("LEGACY MODULE REQUEST HANDLER");
        final GameSession gameSession = this.mGameServerClientConnection.getGameSession();
        if (gameSession != null) {
            final String[] packet = this.mLegacyModuleRequest.message.split("\\|");
            switch (packet[0]) {
                case ServerCommands.SET_STATUS:
                    switch (packet[1]) {
                        case ClientCommands.CONFIGURATION:
                        	gameSession.getPlayer().changeConfiguration(Short.parseShort(packet[2]));
                            break;
                    }
                    break;
            }
        }
    }
}
